nautobot/ipam/api/views.py

```diff
@@ -391,6 +391,51 @@ class VLANGroupViewSet(NautobotModelViewSet):
     serializer_class = serializers.VLANGroupSerializer
     filterset_class = filters.VLANGroupFilterSet
 
+    @action(
+        detail=True,
+        name="Available VLAN IDs",
+        url_path="available-vids",
+        methods=["get"],
```

Comment:
Do we plan to add POST support here in the future for allocating VLANs from this endpoint? If so, I think this should be `available-vlans` rather than `available-vids`, personally.